<launch>
	<!-- global rosparam -->
	<param name="use_sim_time" value="true"/>

	<!-- rosbag -->
	<node pkg="rosbag" type="play" name="player" args="--clock /home/amsl/ozaki/docker_ws/airsim_ros_wrapper/bagfiles/drone_1cam.bag"/>

	<!-- rviz -->
	<node pkg="rviz" type="rviz" name="rviz_perspective" args="-d $(find msg_conversion)/rviz_config/ned_to_neu.rviz"/>

	<!-- MAIN -->
	<node pkg="msg_conversion" type="ned_to_neu" name="ned_to_neu">
		<remap from="/odom" to="/airsim_node/drone/odom_local_ned"/>
		<remap from="/imu/data" to="/airsim_node/drone/imu/Imu"/>
		<!-- <param name="parent_frame_id" type="string" value="/odom"/> -->
		<!-- <param name="child_frame_id" type="string" value="/odom/neu"/> -->
	</node>
</launch>
